client/reader/stream/index.jsx

```diff
@@ -438,6 +501,20 @@ class ReaderStream extends React.Component {
 				/>;
 		}
 
+		if ( postKey.isCombination ) {
+			return <ConnectedCombinedCard
+						postKey={ postKey }
+						index={ index }
+						key={ `combined-card-${ index }` }
+						onClick={ post => showSelectedPost( {
```

Comment:
don't want this to be an anonymous function. it'll cause the combo card to re-render every time we do a render pass, even if nothing changed.